pkg/controller/networkpolicy/antreanetworkpolicy.go

```diff
@@ -118,7 +118,9 @@ func (n *NetworkPolicyController) processAntreaNetworkPolicy(np *crdv1alpha1.Net
 		appliedToGroups = mergeAppliedToGroups(appliedToGroups, atgs...)
 		var peer *controlplane.NetworkPolicyPeer
 		if egressRule.ToServices != nil {
-			peer = n.svcRefToPeerForCRD(egressRule.ToServices, np.Namespace)
+			peer = n.svcRefToPeerForCRD(egressRule.ToServices, np.Namespace, crdv1alpha1.ScopeCluster)
+		} else if n.multiclusterEnabled && egressRule.ToMultiClusterServices != nil {
+			peer = n.svcRefToPeerForCRD(egressRule.ToMultiClusterServices, np.Namespace, crdv1alpha1.ScopeClusterSet)
```

Comment:
Should we add a log somewhere when the `ToMultiClusterServices` is not nil but multi-cluster is not enabled?